adventureGame/AdventureGame/.spyproject/workspace.ini

```diff
@@ -6,5 +6,5 @@ save_non_project_files = False
 
 [main]
 version = 0.1.0
-recent_files = []
+recent_files = ['C:\\Users\\mr_ep\\.spyder-py3\\temp.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\obj\\Inventory.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\test\\test_Inventory.py', 'C:\\Users\\mr_ep\\OneDrive\\Documents\\University\\Software Development\\Assignment 2\\adventureGameRepo\\AdventureGame\\adventureGame\\AdventureGame\\src\\Main.py']
```

Comment:
way too many AdventureGame subfolders, lol

Reply:
Can't help this, it's auto-generated when you load the spyder project file

Reply:
No worries, would be confusing if it was displayed like that on your PC is all.